### PR TITLE
Session management time updates

### PR DESCRIPTION
**Pull request by carltonsmith**
### This pull request changes...

This PR updates the timeouts to the required values documented in PR #86

### CODE REVIEW

File: tdrs-backend/tdpservice/settings/development.py [hunk 1]
```diff
@@ -49,6 +49,3 @@ class Development(Common):
         "all_applications": True,
         "group_models": True,
     }
-
-    # Session Timeout
-    SESSION_TIMEOUT = 5
```
Comment (carltonsmith) [inline tdrs-backend/tdpservice/settings/development.py:54]: Removing this setting causes the development environment to use the timeout in `common.py` which is `30`

File: tdrs-backend/tdpservice/settings/local.py [hunk 2]
```diff
@@ -22,6 +22,3 @@ class Local(Common):
     EMAIL_HOST = "localhost"
     EMAIL_PORT = 1025
     EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
-
-    # Session Timeout
-    SESSION_TIMEOUT = 5
```
Comment (carltonsmith) [inline tdrs-backend/tdpservice/settings/local.py:27]: Removing this setting causes the local environment to use the timeout in `common.py` which is `30`

File: tdrs-frontend/.env.development [hunk 3]
```diff
@@ -1,7 +1,7 @@
 REACT_APP_BACKEND_URL=http://localhost:8080/v1
-REACT_APP_TIMEOUT_TIME=6000
-REACT_APP_LOGOUT_TIME=10000
-REACT_APP_DEBOUNCE_TIME=1000
+REACT_APP_TIMEOUT_TIME=1000 * 60 * 20
+REACT_APP_LOGOUT_TIME=1000 * 60 * 3
+REACT_APP_DEBOUNCE_TIME=1000 * 30
```
Comment (carltonsmith) [inline tdrs-frontend/.env.development:4]: These times are in milliseconds so `1000 * 60 * 20` is 20 minutes, `1000 * 60 * 3` is 3 minutes and `1000 * 30` is 30 seconds. We annotated it this way as it's easier to read than a bunch of zeros.

Comment (alexsoble) [reply tdrs-frontend/.env.development:4]: Great move!

**Comment by alexsoble** [comment]
# QASP Review 

### Deliverable 1: Accepted Features

> *Performance Standard(s): At the beginning of each sprint, the Product Owner and development team will collaborate to define a set of user stories to be completed during the sprint. Acceptance criteria for each story will also be defined. The development team will deliver code and functionality to satisfy these user stories.*

> *Acceptable Quality Level: Delivered code meets the acceptance criteria for each user story. Incomplete stories will be assessed and considered for inclusion in the next sprint.*

+ [ ] Updates session timeout configuration on the backend
+ [ ] Updates session timeout configuration on the frontend

As Product Owner, @lfrohlich will decide if ACs are met.

### Deliverable 2: Tested Code

> *Performance Standard(s): Code delivered under the order must have substantial test code coverage. Version-controlled HHS GitHub repository of code that comprises products that will remain in the government domain.*

> *Acceptable Quality Level: Minimum of 90% test coverage of all code. All areas of code are meaningfully tested.*

_N/A, config change only._

### Deliverable 3: Properly Styled Code

> *Performance Standard(s): GSA 18F Front- End Guide*

> *Acceptable Quality Level: 0 linting errors and 0 warnings*

+ [x] Backend code style checks passing on CircleCI
+ [x] Frontend code style checks passing on CircleCI

### Deliverable 4: Accessible

> *Performance Standard(s): Web Content Accessibility Guidelines 2.1 AA standards*

> *Acceptable Quality Level: 0 errors reported using an automated scanner and 0 errors reported in manual testing*

_N/A, config change only._

### Deliverable 5: Deployed

> *Performance Standard(s): Code must successfully build and deploy into the staging environment.*

> *Acceptable Quality Level: Successful build with a single command*

+ [x] Code successfully deployed via automated CircleCI process to development on Cloud.gov

### Deliverable 6: Documented

> *Performance Standard(s): Summary of user stories completed every two weeks. All dependencies are listed and the licenses are documented. Major functionality in the software/source code is documented, including system diagram. Individual methods are documented inline in a format that permits the use of tools such as JSDoc. All non-inherited 800-53 system security controls are documented in the Open Control or OSCAL format and HHS Section 508 Product Assessment Template (PAT) are updated as appropriate.*

> *Acceptable Quality Level: Combination of manual review and automated testing, if available*

_Config change only._

### Deliverable 7: Secure

> *Performance Standard(s): Open Web Application Security Project (OWASP) Application Security Verification Standard 3.0*

> *Acceptable Quality Level: Code submitted must be free of medium- and high-level static and dynamic security vulnerabilities*

+ [x] OWASP Scan passes on CircleCI

**Review by alexsoble — APPROVED** [review]
LGTM!

**Comment by alexsoble** [comment]
@lfrohlich, if the timeout values look correct to you, I recommend that you merge this PR.

**Pull request merged**